supplementary/alica_ros2/alica_ros2_turtlesim/src/base.cpp

```diff
@@ -90,7 +89,51 @@ Base::~Base()
     spinThread.join();
     ac->terminate();
     delete ac;
-    ALICATurtleWorldModel::del();
+    ALICATurtleWorldModelCallDel();
+}
+
+void Base::ALICATurtleWorldModelCallInit(rclcpp::Node::SharedPtr nh, rclcpp::Node::SharedPtr priv_nh)
+{
+    std::vector<std::string> tmp = calculateLibraryPath();
+    std::string libraryPath = calculateLibraryCompleteName(tmp, "alica-ros2-turtlesim");
+
+    if (libraryPath.empty()) {
+        std::cerr << "Error:"
+                  << "Lib not exists in this path:" << libraryPath << std::endl;
+        return;
+    } else {
+        std::cerr << "Debug:"
+                  << "Lib exists in this path:" << libraryPath << " for ALICATurtleWorldModelInit" << std::endl;
+    }
+
+    typedef void(InitType)(rclcpp::Node::SharedPtr&, rclcpp::Node::SharedPtr&);
+    std::function<InitType> wminit;
```

Comment:
Actually, this should not be necessary I think.  Maybe can copy what is done in alica turtlesim 1 with respect to world model?  World model doesn't need to be dynamically loaded

Reply:
This is more challenging. I  try to do all the modifications.

Reply:
ok